templates/solutions/ai/index.html

```diff
@@ -297,50 +342,61 @@ <h2>Optimised for silicon</h2>
         hi_def=True,
         loading="lazy"
         ) | safe
-      }}
+        }}
+      </div>
     </div>
-  </div>
-</section>
+  </section>
 
-<section class="p-section">
-  <div class="row--50-50">
-    <hr class="p-rule" />
-    <div class="col">
-      <h2>AI at the edge</h2>
-    </div>
-    <div class="col">
-      <p>Drive real-time insight in the field and on the factory floor by running your AI models directly on edge devices.</p>
-      <p>Deploy compute where you need it to power AI use cases at the edge. Solve security, management and update challenges for your fleet with an OS purpose-built for IoT devices and optimised for AI.</p>
-      <hr />
-      <p>
-        <a href="/blog/edge-ai">Edge AI explained&nbsp;&rsaquo;</a>
-      </p>
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>AI at the edge</h2>
+      </div>
+      <div class="col">
+        <p>
+          Drive real-time insight in the field and on the factory floor by running your AI models directly on edge devices.
+        </p>
+        <p>
+          Deploy compute where you need it to power AI use cases at the edge. Solve security, management and update challenges for your fleet with an OS purpose-built for IoT devices and optimised for AI.
+        </p>
+        <hr />
+        <p>
+          <a href="/engage/guide-to-deploying-ai-at-the-edge">Edge AI explained&nbsp;&rsaquo;</a>
```

Comment:
Change here

Reply:
This link seems to be 404: https://canonical-com-1246.demos.haus/engage/guide-to-deploying-ai-at-the-edge

Reply:
I think the link is on ubuntu.com, not canonical.